GameServerLib/API/ApiFunctionManager.cs

```diff
@@ -118,9 +118,41 @@ public static bool IsWalkable(float x, float y)
 
         public static void AddBuff(string buffName, float duration, byte stacks, BuffType buffType, IObjAiBase onto, IObjAiBase from)
         {
+            if (duration <= 0)
+            {
+                return;
+            }
+
             var buff = new Buff(_game, buffName, duration, stacks, buffType, onto, from);
+
             onto.AddBuff(buff);
+
             _game.PacketNotifier.NotifyAddBuff(buff);
+
+            CreateTimer(duration, () =>
```

Comment:
Add here `// HACK: Remove timers and use proper tick system to handle the buffs using priority queue of duration and order`

Reply:
Added.